src/assets/scss/_overwrite-mat.scss

```diff
@@ -81,7 +81,7 @@ app-campaign-card {
   padding: 0.3em 0 0.7em 0;
 }
 
-.mat-form-field-appearance-legacy .mat-form-field-infix {
+.donation-input .mat-form-field-appearance-legacy .mat-form-field-infix {
```

Comment:
@tbg-jasonfung can these not just apply to the specific fields and live on the component instead of in the global file?